### PR TITLE
Login in Yandex test was added

### PR DESCRIPTION
**Pull request by igorchupin**
Login in Yandex test was added
By variables are created in the separate class

### CODE REVIEW

File: src/test/java/LoginInYandexTest.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    private WebDriver driver;
+    private String username = "firstnamelastname1989";
+    private String password = "pTKJctHpbaj@t7M";
+    private String expectedURL = "https://mail.yandex.com/?uid=1459838850#tabs/relevant";
+    private String expectedName = "Inbox — Yandex.Mail";
+
+
+    @BeforeAll
+    public static void beforeClass () {
+
```
Comment (EugenBorisik) [inline src/test/java/LoginInYandexTest.java]: Remove blank line

File: src/test/java/LoginInYandexTest.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    @AfterEach
+    public void afterTest () {
+        driver.quit();
+    }
+
+
+    @Test
+    @Tag("yandex_login_test")
+
```
Comment (EugenBorisik) [inline src/test/java/LoginInYandexTest.java]: Remove blank line

File: src/test/java/LoginInYandexTest.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    void yandexLoginTest () {
+
```
Comment (EugenBorisik) [inline src/test/java/LoginInYandexTest.java]: Remove blank line

File: src/test/java/LoginInYandexTest.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        WebElement enterButton = driver.findElement(By.xpath("//*[@id=\"index-page-container\"]/div/div[2]/div/div/div[4]/a[2]"));
+        enterButton.click();
+
+        WebElement emailField = driver.findElement(By.id("passp-field-login"));
+        emailField.sendKeys(username);
+        WebElement loginButton = driver.findElement(By.id("passp:sign-in"));
+        loginButton.click();
+
+        WebElement passwordField = driver.findElement(By.id("passp-field-passwd"));
+        passwordField.sendKeys(password + "\n");
```
Comment (EugenBorisik) [inline src/test/java/LoginInYandexTest.java]: I don't like this "\n" in the end, please remove

**Review by EugenBorisik — CHANGES_REQUESTED** [review]

File: src/test/java/LoginInYandexTest.java (hunk 1 above)
Comment (igorchupin) [reply src/test/java/LoginInYandexTest.java]: Removed

File: src/test/java/LoginInYandexTest.java (hunk 2 above)
Comment (igorchupin) [reply src/test/java/LoginInYandexTest.java]: Removed

File: src/test/java/LoginInYandexTest.java (hunk 3 above)
Comment (igorchupin) [reply src/test/java/LoginInYandexTest.java]: removed

File: src/test/java/LoginInYandexTest.java (hunk 4 above)
Comment (igorchupin) [reply src/test/java/LoginInYandexTest.java]: Changed to id locator

**Review by EugenBorisik — APPROVED** [review]

**Pull request merged**